messages/build_depends.repos

```diff
@@ -0,0 +1,9 @@
+repositories:
+  msgs/autoware_api_msgs:
+    type: git
+    url: https://github.com/tier4/AutowareArchitectureProposal_api_msgs.git
```

Comment:
Could you move packages from this repository to `autoware.universe/messages/`?

Reply:
create PR https://github.com/autowarefoundation/autoware.universe/pull/20

Reply:
Cool!